frappe/website/doctype/blog_post/blog_post.py

```diff
@@ -126,6 +128,11 @@ def get_list_context(context=None):
 	list_context.update(frappe.get_doc("Blog Settings", "Blog Settings").as_dict(no_default_fields=True))
 	return list_context
 
+def get_featured_doc():
+	featured = frappe.db.get_single_value('Website Settings', 'featured_blog')
+	featured_doc = frappe.get_doc("Blog Post", featured)
+	return featured_doc
```

Comment:
```suggestion
	featured = frappe.db.get_single_value('Website Settings', 'featured_blog')
	featured_doc = frappe.get_doc("Blog Post", featured)
	return featured_doc
```

Can we make featured blog check inside a blog rather than `Website Settings` ?

Reply:
This was @neilLasrado's take 
@neilLasrado How do u want it

Reply:
Only one blog can be a featured blog. Website settings still might be a better place? what do you think?